src/views/Swap/hooks/useWarningImport.tsx

```diff
@@ -37,7 +37,7 @@ export default function useWarningImport() {
   const importTokensNotInDefault =
     urlLoadedTokens &&
     urlLoadedTokens.filter((token: Token) => {
-      return !(token.address in defaultTokens) && token.chainId === chainId
+      return !(token.address in defaultTokens) && token.chainId === chainId && !isWrongNetwork
```

Comment:
if user is in wrong network, can se simply return empty array instead of calling filter